content/wasm-functions/using-cron-jobs.md

```diff
@@ -0,0 +1,237 @@
+```console
+App 'hello-cron-job' initialized successfully.
+Waiting for application to be ready... ready                                                                                   
+
+Application URL:   https://1064f8cc-4b83-4fb0-a550-ff51d8a85a18.aka.fermyon.tech/
+  Routes:
+  - hello-cron-job: https://1064f8cc-4b83-4fb0-a550-ff51d8a85a18.aka.fermyon.tech/cron-me
+```
+
+You can use the domain name provided to you by Fermyon Wasm Functions to test that the Spin application is working as expected by curling the new endpoint:
+
+<!-- @selectiveCpy -->
+
+```console
+curl https://1064f8cc-4b83-4fb0-a550-ff51d8a85a18.aka.fermyon.tech/cron-me
+```
+
+You should receive the same message as you did during local testing:
+
+<!-- @nocpy -->
+
+```console
+Cron job executed
+```
+
+# Scheduling a Cron Job for Your Spin Application
+
+Now we’ll use the `spin aka crons` command to invoke the Spin application's HTTP endpoint on a scheduled basis. The schedule uses standard crontab syntax.
+
+> `spin aka crons` supports multiple digits in each cron field, as well as intervals in any position (e.g., `* */12 * * *` to run every 12 hours). You can also use comma-separated lists and ranges. If your schedule includes a specific time of day, be sure to use UTC—you may need to [convert your local time to UTC](https://www.worldtimebuddy.com/?pl=1&lid=100&h=100&hf=1) to ensure correct execution.
+
+Let’s create a cron job that triggers your Spin application every 5 minutes. The `spin aka crons` create subcommand takes three arguments:
+
+* **Schedule** – How often the job runs, in standard cron syntax
+* **Path and query** – The HTTP path (and optional query parameters) to invoke
+* **Name** – A name for your cron job (this will be associated with the Spin application in your current working directory)
+
+In this example, we’ll schedule a job to hit the `/cron-me` path with a msg query parameter set to "fwf" every 5 minutes, for a Spin app named hello-cron-job.
+
+<!-- @selectiveCpy -->
+
+```console
+spin aka crons create "*/5 * * * *" "/cron-me?msg=fwf" "cron-job-1"
+```
+
+You should see output similar to:
+
+<!-- @nocpy -->
+
+```console
+Created cron which will next run at 2025-04-02 04:00:00 UTC
+```
+
+If we wait 5 minutes and check our application logs with the `spin aka logs` subcommand, we will see that our cron job invoked the application every five minutes as expected. 
+
+<!-- @nocpy -->
+
+```console
+spin aka logs -a hello-cron-job
+2025-04-02 04:00:00 [hello-cron-job]  Cron job triggered at 2025-04-12T04:00:00Z with msg: "fwf" 
+```
+
+# Managing Cron Jobs
+
+You can view all of your running cron jobs with:
+
+<!-- @selectiveCpy -->
+
+```console
+spin aka crons list
+```
+
+<!-- @nocpy -->
+
+```console
++----------------+--------------+-------------------------+
+| Name           | Schedule     | Next Run                |
++=========================================================+
+| hello-cron-job |  */5 * * * * | 2025-04-02 04:00:00 UTC |
```

Comment:
```suggestion
| cron-job-1 |  */5 * * * * | 2025-04-02 04:00:00 UTC |
```

or wait, are you saying this is the app name rather than the job name?

(I am not sure if the command shows _all_ cron jobs or only the ones associated with the current app.)

Reply:
d'oh - typo! I fixed this. I also renamed the app to "hello-world" to avoid confusion between app name and cron job name.  

> I am not sure if the command shows all cron jobs or only the ones associated with the current app.)
@rylev any guidance on this? I'm not sure

Reply:
Having thoroughly distinct different names for the app and the cron job is an excellent idea - thank you!